pkg/operator/periodicbackupcontroller/periodicbackupcontroller.go

```diff
@@ -318,3 +334,114 @@ func newCronJob() (*batchv1.CronJob, error) {
 
 	return obj.(*batchv1.CronJob), nil
 }
+
+func deployBackupServerDaemonSet(cr backupv1alpha1.Backup, endpoints []string) *appv1.DaemonSet {
```

Comment:
this function does not deploy anything, it only returns a DS

Reply:
+1 changing 👍🏽